### PR TITLE
build-iso: add computation of the sha256 checksum

### LINKED ISSUE #228: Generate checksum file after building ISO
luet-makeiso had a convenience functionality to generate sha256 checksum files directly after build. 

While this can be easily workarounded after the build process, this was something nice to have to have the build process secured in a homogeneous way.

### PR DESCRIPTION
**Pull request by fgiudici**
Add sha256 file when building the iso.

Fixes #228

### CODE REVIEW

**Comment by fgiudici** [comment]
(rebased on main)

File: pkg/action/build-iso.go [hunk 1]
```diff
@@ -234,6 +236,22 @@ func (b BuildISOAction) burnISO(root string) error {
...
+	// When we start Xorriso we expect either building the ISO img file or returning
+	// an error. During tests anyway we use the FakeRunner Runner: the Xorriso cmd is
+	// not run and we land here with no ISO img file and no error.
+	// Let's skip the checksum computation if the ISO img file is missing.
+	if exists, _ := utils.Exists(b.cfg.Fs, outputFile); exists {
+		checksum, err := utils.CalcFileChecksum(b.cfg.Fs, outputFile)
+		if err != nil {
+			return fmt.Errorf("checksum computation failed: %w", err)
+		}
+		err = b.cfg.Fs.WriteFile(outputFile+".sha256", []byte(fmt.Sprintf("%s %s\n", checksum, isoFileName)), 0644)
```
Comment (mudler) [inline pkg/action/build-iso.go]:
```suggestion
		err = b.cfg.Fs.WriteFile(fmt.Sprintf("%s.sha256",outputFile)), []byte(fmt.Sprintf("%s %s\n", checksum, isoFileName)), 0644)
```

Comment (mudler) [reply pkg/action/build-iso.go]: just for consistency :)

**Review by mudler — APPROVED** [review]
Just a small nit, looks good here!

**Pull request merged**